experimental/FTheoryTools/src/WeierstrassModels/constructors.jl

```diff
@@ -69,7 +69,8 @@ function weierstrass_model(base::NormalToricVariety,
   set_coordinate_names(fiber_ambient_space, ["x", "y", "z"])
   D1 = 2 * anticanonical_divisor_class(base)
   D2 = 3 * anticanonical_divisor_class(base)
-  ambient_space = _ambient_space(base, fiber_ambient_space, D1, D2)
+  D3 = trivial_divisor_class(base)
+  ambient_space = _ambient_space(base, fiber_ambient_space, [D1, D2, D3])
   
   # construct the model
   pw = _weierstrass_polynomial(explicit_model_sections["f"], explicit_model_sections["g"], cox_ring(ambient_space))
```

Comment:
This is an aside, but the fact that the WeierstrassModel type takes in the entire hypersurface polynomial as input, rather than f and g, means that in principle the user could make a WeierstrassModel object that has a completely different hypersurface equation, and then presumably all of our functions related to Weierstrass models would fail. This is maybe something we should think about in the future.